dist/goldenlayout.js

```diff
@@ -4165,7 +4183,7 @@ lm.utils.copy( lm.items.RowOrColumn.prototype, {
 	 */
 	_createSplitter: function( index ) {
 		var splitter;
-		splitter = new lm.controls.Splitter( this._isColumn, this._splitterSize );
+		splitter = new lm.controls.Splitter( this._isColumn, this._splitterSize, this._splitterGrabSize );
```

Comment:
this also looks like a confused merge ... or perhaps I'm misreading the github ui?